.github/workflows/e2e-test.yml

```diff
@@ -21,16 +21,35 @@ env:
   REPO_NAME_DOCKER: docker-hosted
   REPO_NAME_PYPI: pypi-hosted
 
-
 jobs:
+  filter:
```

Comment:
e2e-test is only running on the main branch by crontab, so there is no need to config filter.